_episodes/05-cond.md

```diff
@@ -299,6 +303,10 @@ freeing us from having to manually examine every plot for features we've seen be
 >     print('not not True is true')
 > ~~~
 > {: .language-python}
+> > ## Solution
+> > Again, empty string is not true, and non-empty string is true.
+> > Not (not true, i.e. false) is true. 
+> {: .solution}
```

Comment:
Um, I'm confused. The exercise says:

> After reading and running the code below,
> write some `if` statements that use `not` to test the rule
> that you formulated in the previous challenge.

Reply:
Should we add a solution such as:

if not not '':
    print('empty string is true')
if not not 'word':
    print('word is true')
if not not []:
    print('empty list is true')
if not not [1, 2, 3]:
    print('non-empty list is true')
if not not 0:
    print('zero is true')
if not not 1:
    print('one is true')

Reply:
The task is to
> write some `if` statements that use `not` to test the rule
> that you formulated in the previous challenge.

So, we have to formulate the rule first.
If the rule is
> A string is "true" as long as it contains at least one symbol.
> A list is "true" as long as it contains at least one element.
> Any non-zero number is "true".

Then we could write:
> if not 0:
>    print('not 0 is true')
> if not '0':
>     print('this will not print because our string is not empty')
> if not []:
>     print('not empty list is true')
> if not [[]]:
>     print('this will not print because our list has one element - another empty list')

Reply:
Yes I agree. And it would be much clearer!